package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.5.1",
   "description": "",
   "keywords": [],
-  "author": "",
+  "author": "Walmart Labs <http://walmartlabs.com/>",
```

Comment:
Wasn't sure if there's an email or preferred electrode author here. We should add [a `contributors` field](https://docs.npmjs.com/files/package.json#people-fields-author-contributors) to list all the folks!

Reply:
This top level `package.json` is not actually used for publication, we don't publish this module on NPM, it's just the top level `package.json` mostly for lerna (multi package setup). All `ern-*` packages are getting published and the contributors field is there for each of them in their `package.json`. 
For this one we should just remove the author, unless there is a good reason to use this field (for all `ern-*` modules we just removed this field). I feel like it's good to use this field if the author is a real person, not an org.
FYI all packages get published using `electrode-native` user on GitHub https://www.npmjs.com/~electrode-native, it also has the email `electrodenative@gmail.com` ... So in case we use an author we should use this email and username. But I feel it's not really needed here, lets just remove it.

Reply:
Totally makes sense! We should maybe add `"private": true` to the root `package.json` to make sure nobody accidentally publishes.Totally makes sense! We should maybe add `"private": true` to the root `package.json` to make sure nobody accidentally publishes.

Reply:
Totally makes sense! We should maybe add `"private": true` to the root `package.json` to make sure nobody accidentally publishes.